test/aerosol_activation_emulators.jl

```diff
@@ -0,0 +1,362 @@
+                    transformer = target_transform,
+                    inverse = inverse_target_transform,
+                )
+        elseif ml_model == "ET"
+            pipeline =
+                _preprocess_aerosol_data |>
+                Standardizer() |>
+                MLJ.TransformedTargetModel(
+                    MLJ.TunedModel(
+                        tuning = MLJ.Grid(goal = 30),
+                        model = EvoTreeRegressor(),
+                        resampling = MLJ.CV(nfolds = 5),
+                        range = [
+                            range(
+                                EvoTreeRegressor(),
+                                :eta,
+                                lower = 0.05,
+                                upper = 1,
+                                scale = :log,
+                            ),
+                            range(
+                                EvoTreeRegressor(),
+                                :max_depth,
+                                lower = 3,
+                                upper = 15,
+                            ),
+                        ],
+                    ),
+                    transformer = target_transform,
+                    inverse = inverse_target_transform,
+                )
+        else
+            error("Unknown ML method!")
+        end
+
+        # Create the untrained ML model
+        mach = MLJ.machine(pipeline, X_train, Y_train)
+        # Train a new ML model
+        @info(
+            "Training a new ML model ($(ml_model)). This may take a minute or two."
+        )
+        MLJ.fit!(mach, verbosity = 0)
+        # Save the ML model to a binary file that can be re-used by CloudMicrophysics
+        MLJ.save(joinpath(fpath, machine_name), mach)
+        return mach
+    end
+end
+
+function test_emulator(FT)
+
+    aip = CMP.AirProperties(FT)
+    tps = TD.Parameters.ThermodynamicsParameters(FT)
+    ap = CMP.AerosolActivationParameters(FT)
+
+    # Atmospheric conditions
+    T = FT(294)    # air temperature K
+    p = FT(1e5)    # air pressure Pa
+    w = FT(0.5)    # vertical velocity m/s
+    p_vs = TD.saturation_vapor_pressure(tps, T, TD.Liquid())
+    q_vs = 1 / (1 - TD.Parameters.molmass_ratio(tps) * (p_vs - p) / p_vs)
+    q = TD.PhasePartition(q_vs)
+
+    # Aerosol size distribution
+    salt = CMP.Seasalt(FT)
+    # Accumulation mode
+    r1 = FT(0.243 * 1e-6) # m
+    σ1 = FT(1.4)          # -
+    N1 = FT(100 * 1e6)    # 1/m3
+    # Coarse Mode
+    r2 = FT(1.5 * 1e-6)   # m
+    σ2 = FT(2.1)          # -
+    N2 = FT(1e6)          # 1/m3
+    acc = AM.Mode_κ(r1, σ1, N1, (FT(1.0),), (FT(1.0),), (salt.M,), (salt.κ,), 1)
+    crs = AM.Mode_κ(r2, σ2, N2, (FT(1.0),), (FT(1.0),), (salt.M,), (salt.κ,), 1)
+    ad = AM.AerosolDistribution((crs, acc))
+
+    # NN
+    # Get the ML model
+    mach_NN = get_2modal_model_FT32(
+        ml_model = "NN",
+        machine_name = "2modal_nn_machine_naive.jls",
+    )
+
```

Comment:
Could we encapsulate individual tests with 

```
TT.@testset "0M_microphysics" begin

end
```
blocks? I think this way they display nicer in the test summary in the CI